ihtml/themes/breezy/login.tpl

```diff
@@ -1,7 +1,6 @@
 <body>
 
   {$php_errors}
-  {include file={filePath file="setup_header.tpl"}}
```

Comment:
Why?
This removes logo and version if I’m not mistaking?
I get that you move version to the copyright notice, but the logo should be there IMO.